src/coreComponents/physicsSolvers/NonlinearSolverParameters.cpp

```diff
@@ -111,6 +113,8 @@ NonlinearSolverParameters::NonlinearSolverParameters( string const & name,
     setInputFlag( InputFlags::OPTIONAL ).
     setDescription( "Max number of times that the configuration can be changed" );
 
+  registerWrapper( viewKeysStruct::totalSuccessfulNewtonNumIterationsString, &m_totalSuccessfulNewtonNumIterations );
+  registerWrapper( viewKeysStruct::totalWastedNewtonNumIterationsString, &m_totalWastedNewtonNumIterations );
```

Comment:
If you use one class instead of multiple integers you'll be able to register it once.